multi-node/aws/.gitignore

```diff
@@ -1,4 +1,5 @@
 /artifacts/template.json
 /bin
-/clusters
-/cluster.yaml
+/assets
+*.yaml
+*~
```

Comment:
the *~ is actually what i want there. emacs by default suffixes ~ for autosave files